### ISSUE TITLE
remove core-util-is

### ISSUE THREAD

**Issue by mcollina** [pinojs/pino#150]
`util.is*` methods are in fact semi-deprecated in core, and we do not encourage to use them, so why should we?

**Comment by jsumners** [comment]
@mcollina if you were going on `core-util-is` being in our dependency list, we weren't even using that module. PR #151 removes it. Other than that, I don't see anywhere in our code base that even has a `util.is`.

By the way, `core-util-is` still gets installed with that PR because of this dep tree:

```
% npm install                       
(node:50062) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
pino@3.0.5 /Users/jsumners/Projects/pino
└─┬ flush-write-stream@1.0.0
  └─┬ readable-stream@2.1.5
    └── core-util-is@1.0.2
```

**Comment by davidmarkclements** [comment]
I was perplexed here - I thought we weren't using it. 100% agree on removal

**Comment by mcollina** [comment]
I just spot that on our `package.json`. I know readable-stream uses it. Long story. :D

**Comment by jsumners** [comment]
Do we consider this resolved by #151?

**Comment by mcollina** [comment]
Yes!!
